inst/rmd/h/default.html

```diff
@@ -529,7 +532,10 @@ <h4 class="date">$date$</h4>
 $else$
 $if(toc)$
 <div id="$idprefix$TOC">
-$toc$
+$if(toc-title)$
+<h2 id="$idprefix$toc-title">$toc-title$</h2>
+$endif$
+$table-of-contents$
```

Comment:
The variable `table-of-contents` was introduced in Pandoc 2.0: https://github.com/jgm/pandoc/blob/master/changelog.md#pandoc-20-2017-10-29 We need to wait a little longer before changing `toc` to `table-of-contents`.

```suggestion
$toc$
```